cmd/core/main.go

```diff
@@ -112,6 +112,8 @@ func main() {
 	flag.StringVar(&storageDriver, "storage-driver", "Local", "Application file save to the storage driver")
 	flag.DurationVar(&syncPeriod, "informer-re-sync-interval", 60*time.Minute,
 		"controller shared informer lister full re-sync period")
+	flag.DurationVar(&commonconfig.ReconcileTimeout, "reconcile-timeout", time.Minute,
```

Comment:
shall we make it longer since we already know addon will take long time.

Reply:
what about 3min

Reply:
good idea